src/_jq-processor.ts

```
@@ -0,0 +1,97 @@
+ * @returns The processed output from jq
+ */
+export async function processWithJq(jsonData: unknown, jqCommand: string): Result.ResultAsync<string, Error> {
+	// Convert JSON data to string
+	const jsonString = JSON.stringify(jsonData);
+
+	// Use Result.try with object form to wrap spawn call
+	const result = Result.try({
+		try: async () => {
+			const spawnResult = await spawn('jq', [jqCommand], {
+				stdin: jsonString,
+			});
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)